Code/Framework/AzQtComponents/AzQtComponents/Tests/FloatToStringConversionTests.cpp

```diff
@@ -64,3 +64,43 @@ TEST(AzQtComponents, FloatToString_TruncateShowThousandsSeparatorOnlyOneDecimal)
     int numDecimalPlaces = 2;
     EXPECT_EQ(AzQtComponents::toString(1000.000, numDecimalPlaces, testLocal, showThousandsSeparator), "1,000.0");
 }
+
+TEST(AzQtComponents, FloatToString_Truncate2DecimalsWithLocale)
+{
+    QLocale testLocal{ QLocale() };
```

Comment:
Nit: Shouldn't all of these be "testLocale" instead of "testLocal"?

Reply:
Indeed, will fix.

Reply:
WRT, the serialization, I will try to create some tests tomorrow to see if this is an issue.